packages/orchestration/src/exos/chain-hub.js

```diff
@@ -149,18 +148,18 @@ const ChainHubI = M.interface('ChainHub', {
 });
 
 /**
- * Make a new ChainHub in the zone (or in the heap if no zone is provided).
+ * Make a new ChainHub in the zone.
  *
  * The resulting object is an Exo singleton. It has no precious state. It's only
  * state is a cache of queries to agoricNames and whatever info was provided in
  * registration calls. When you need a newer version you can simply make a hub
  * hub and repeat the registrations.
```

Comment:
```suggestion
 * registration calls. When you need a newer version you can simply make a ???
 * hub and repeat the registrations.
```

"hub hub" is clearly wrong. But I'm not sure if you meant "new hub", which would be distinct from repeating the registrations in a new incarnation of the "same" hub. If a new hub, how does it get wired in to uses of the old hub?